.github/workflows/release_linux.yml

```diff
@@ -0,0 +1,59 @@
+name: Release Linux
+
+on: 
+  workflow_dispatch:
+    inputs:
+      commit_hash:
+        description: "Release Candidate Commit (example: 5a3ae3)"
+        required: true
+
+jobs:
+  release: 
+    env:
+      GITHUB_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2.3.4
+      with: 
+        commit: ${{ github.event.inputs.commit_hash }}
+
+    - name: Install Node
+      uses: actions/setup-node@v2.2.0
+      with:
+        node-version: 16
+
+    - name: Install NPM Dependencies
+      run: npm ci
+
+    - name: Tag Release Commit
+      run: |
+        git config --global user.name "OutlineBot"
+        git config --global user.email "outlinebot@users.noreply.github.com"
+
+        git tag "linux-$(node scripts/get_version.mjs linux)"
+
+    - name: Download Release Candidate
+      uses: dawidd6/action-download-artifact@v2
+      with:
+        workflow: build_release_candidates.yml
+        commit: ${{ github.event.inputs.commit_hash }}
+        files: |
+          build/dist/Outline-Client.AppImage
+          build/dist/linux-beta.yml
+
+    - name: Checkout Releases Repository
+      uses: actions/checkout@v2.3.4
+      with:
+        repository: ${{ env.RELEASES_REPOSITORY }}
+        token: ${{ env.GITHUB_TOKEN }}
+        path: outline-releases
+    
+    - name: Push to Releases Repository
+      run: |
+        mv build/dist/Outline-Client.AppImage outline-releases/client/stable/Outline-Client.AppImage
+        mv build/dist/linux-beta.yml outline-releases/client/stable/linux-latest.yml
+        RELEASE_VERSION="$(node scripts/get_version.mjs linux)""
+        cd outline-releases
+        git add client/stable/Outline-Client.AppImage client/stable/linux-latest.yml
+        git commit -m "release linux client ${RELEASE_VERSION}"
```

Comment:
Can we make sure that the client is clean and there are no other edited non-committed file?
I want to avoid mistakes where we modify a file by mistake.

Reply:
Yep, I'll confirm this in my testing today.

Reply:
(Sorry, testing this is next on my list right after confirming the android build fix)